mission_control/src/mission_control.cpp

```diff
@@ -55,6 +55,8 @@ MissionControlNode::MissionControlNode(ros::NodeHandle& h) : pnh(h)
   pnh.getParam("report_heart_beat_rate", reportHeartbeatRate);
   pnh.getParam("execute_mission_asynchronous_rate", executeMissionAsynchronousRate);
 
+  pnh.param<double>("/fin_control/max_ctrl_fin_angle", maxCtrlFinAngle, 10.0);
```

Comment:
@patapufate meta: I'd suggest we leave `/fin_control/` out of the parameter name, to later introduce it via remapping on launch.

Reply:
I'm taking from the parameter names same as the autopilot does. How should I Implement this?